cmd/crates/soroban-test/tests/it/integration/dotenv.rs

```diff
@@ -1,67 +1,68 @@
 use soroban_test::TestEnv;
 
-use super::util::{deploy_hello, TEST_CONTRACT_ID};
+use super::util::deploy_hello;
 
 fn write_env_file(e: &TestEnv, contents: &str) {
     let env_file = e.dir().join(".env");
-    std::fs::write(&env_file, contents).unwrap();
+    let contents = format!("SOROBAN_CONTRACT_ID={contents}");
+    std::fs::write(&env_file, &contents).unwrap();
     assert_eq!(contents, std::fs::read_to_string(env_file).unwrap());
 }
 
-fn contract_id() -> String {
-    format!("SOROBAN_CONTRACT_ID={TEST_CONTRACT_ID}")
+#[tokio::test]
+async fn can_read_file() {
+    let e = &TestEnv::new();
+    std::thread::sleep(core::time::Duration::from_millis(1000));
```

Comment:
Do you have a sense of what's causing the need for this delay? Is it waiting for the quickstart image to start up before deploying, or something else?

Reply:
Nope the health check ensures it's ready and the test has funded an account by this point. So still a bit stumped. Will get more logging info tomorrow.

Reply:
Got it. Feel free to merge and follow up.

Reply:
https://github.com/stellar/soroban-cli/issues/1231